internal/charmstore/addentity.go

```diff
@@ -599,7 +600,7 @@ func (s *Store) bundleCharms(ids []string) (map[string]charm.Charm, error) {
 			// be returned to the user along with other bundle errors.
 			continue
 		}
-		e, err := s.FindBestEntity(url, StableChannel, map[string]int{})
+		e, err := s.FindBestEntity(url, mongodoc.StableChannel, map[string]int{})
```

Comment:
This should match the scheme we used in addRequiredCharms which is mongodoc.NoChannel